### DISCUSSION TITLE
Theme just for GUI Neovide

### DISCUSSION THREAD

**Discussion by ghost** [neovide/neovide#1018] [Q&A] [answered]
It is possible to set theme just for neovide, not neovim in console?? I'm using linux btw.

**Comment by dineshKumar777** [comment, marked as answer]
I haven't tried this personally but you can mention required theme inside if condition (g:neovide) as mentioned in https://github.com/neovide/neovide/discussions/919#discussioncomment-1241138

**Reply by Kethku** [reply]
This should work just fine. It gets set before your config is loaded

**Reply by ghost** [reply]
Yeah! It's just works 💪
``` vim
if exists('g:neovide')
	packadd! dracula
	syntax enable
	colorscheme dracula
en
```
